test/protractor.conf.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
 const isMac = /^darwin/.test(process.platform);
-let isDocker = require('is-docker')();
+// const isDocker = require('is-docker')();
 const { SpecReporter } = require('jasmine-spec-reporter');
 const HtmlScreenshotReporter = require('protractor-jasmine2-screenshot-reporter');
 const jasmineReporters = require('jasmine-reporters');
-
+// const SERVICE_ACCOUNT_TOKEN = process.env.SERVICE_ACCOUNT_TOKEN;
```

Comment:
Is it ok to keep this commented out ? Not really sure since the var is also defined in environment.js :))